acceptance-tests/globalSetup.ts

```diff
@@ -0,0 +1,29 @@
+import { existsSync, mkdirSync, rmSync } from 'fs';
```

Comment:
This is a global setup module, so the setup and teardown methods will only run one time a piece